npm_and_yarn/helpers/lib/yarn/index.js

```diff
@@ -0,0 +1,11 @@
+const lockfileParser = require("./lockfile-parser");
+const updater = require("./updater");
+const subdependencyUpdater = require("./subdependency-updater");
+const peerDependencyChecker = require("./peer-dependency-checker");
+
+module.exports = {
+  parseLockfile: lockfileParser.parse,
+  update: updater.updateDependencyFiles,
+  updateSubdependency: subdependencyUpdater.updateDependencyFile,
+  checkPeerDependencies: peerDependencyChecker.checkPeerDependencies
```

Comment:
Would rather make these include the package manager so we can require these explicitly, something like:

```js
module.exports = {
  'yarn:parseLockfile': lockfileParser.parse,
  'yarn:update': updater.updateDependencyFiles,
  'yarn:updateSubdependency': subdependencyUpdater.updateDependencyFile,
  'yarn:checkPeerDependencies': peerDependencyChecker.checkPeerDependencies
}
```